tests/test_did_peer_4.py

```diff
@@ -41,12 +49,15 @@
 
 
 def test_encode_decode():
+    encoded = encode(DOC, validate=False)
     encoded = encode(DOC)
```

Comment:
Is there any reason why we're encoding twice and throwing away the results after saving them to the encoded variable?

Reply:
Just to exercise more paths through the code in that test.

Reply:
That one line increased coverage by a whole 1-2% :laughing: